doc/crypto/DSA_sign.pod

```diff
@@ -31,6 +31,10 @@ in newly allocated B<BIGNUM>s at *B<kinvp> and *B<rp>, after freeing
 the old ones unless *B<kinvp> and *B<rp> are NULL. These values may
 be passed to DSA_sign() in B<dsa-E<gt>kinv> and B<dsa-E<gt>r>.
 B<ctx> is a pre-allocated B<BN_CTX> or NULL.
+The precomputed values from DSA_sign_setup() B<MUST NOT be used> for
+more than one signature: using twice the same B<dsa-E<gt>kinv> and
+B<dsa-E<gt>r> pair under the same private key on different plaintexts
```

Comment:
This doesn't quite read right to me. I would put "twice" after "pair"

Reply:
fixed